### PR TITLE
Ticket 6

### PR DESCRIPTION
**Pull request by OwenPortfolio**
Add individual article pages

### CODE REVIEW

File: src/components/Article.jsx [hunk 1]
```diff
@@ -1,8 +1,37 @@
...
+
+    const {article} = useParams();
+    const [currentArticle, setCurrentArticle] = useState();
+    const [loading, setLoading] = useState(true);
+
+    useEffect(() => {
+        getArticleById(article).then((articleById) => {
+            setCurrentArticle(articleById.article);
+            setLoading(false);
+        })
```
Comment (DCrawley94) [inline src/components/Article.jsx:15]: Nicely done!

**Review by DCrawley94 — CHANGES_REQUESTED** [review]
Looking good, but try and refactor your routes in App.js :)

**Review by DCrawley94 — CHANGES_REQUESTED** [review]
Looking good, but try and refactor your routes in App.js :)

File: src/App.js [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
+import { BrowserRouter, Routes, Route} from 'react-router-dom';
 
 import Header from './components/Header'
 import Navbar from './components/Navbar'
 import Body from './components/Body'
 import SingleTopic from './components/SingleTopic'
+import Article from './components/Article'
 
 function App() {
   const [currentPage, setCurrentPage] = useState('All');
```
[the remark this answers is not in the document]
Comment (OwenPortfolio) [reply src/App.js:13]: A leftover! Got it out.

**Pull request merged**